### ISSUE TITLE
[SPIKE] Develop prioritized list of Filters for Platform

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#13645]
## Problem statement
As Platform team works to refactor the filtering, it would be beneficial to know what are the different potential use cases from an Engagement perspective to help prioritize the filter "components" they are proposing.

## What you need to know
- [SRD-Routing](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B3A2FA783-E8DC-4E9B-8397-DA9F5D968DFA%7D&file=SRD-Routing.docx&nav=eyJjIjoxNjA1NzM2NDg0fQ&action=default&mobileredirect=true)

## Acceptance criteria
- [ ] Onboarding engineers document all common filter use cases they can think of with examples
- [ ] Onboarding engineers prioritize common filter use cases
- [ ] Share list with Platform team and get alignment

**Comment by chris-kuryak** [comment]
Moving out one sprint.
Making dependent on #13065

**Comment by chris-kuryak** [comment]
Ideally, this would be the process
- Start with a list of filters we are using in the UP and CP
- Separate by Receiver
- Look to see how frequently they are used

But we don't think that "componentizing" FHIR Path is a high priority.

Benefits of componentizing FHIR Path:
- Save onboarding engr time - THERE IS NOT A LOT OF TIME SAVINGS HERE, NOT WORTH 
- Help prevent errors in creating filters - THIS IS NOT THE BEST SOLUTION FOR THIS
- This will make it easier for end users - Do we need to do this now? We do not envision end-users being able to use this anytime soon

Better solution for "helping prevent errors in creating filters":
- Better/more automated internal testing tools when creating filters
- More enforced testing process

**Comment by chris-kuryak** [comment]
We can come back to this in the future

**Comment by chris-kuryak** [comment]
Checking with Arnej in Slack 2/28

**Comment by arnejduranovic** [comment]
This ticket is outdated and no longer needed. The new direction for filters, as implemented by #16142, makes this ticket moot.
